src/sparseml/pytorch/utils/quantization/quantize_qat_export.py

```diff
@@ -593,3 +599,65 @@ def quantize_torch_qat_export(
         onnx.save(model, output_file_path)
 
     return model
+
+
+def _skip_input_quantize(model: ModelProto) -> bool:
+    if len(model.graph.input) != 1:
+        return False
+    if (
+        len(model.graph.input) != 1
+        or model.graph.input[0].type.tensor_type.elem_type != 1
+    ):
+        # more than 1 input or input is not FP32
+        return False
+
+    input_node = model.graph.input[0]
+    input_children = [
+        node for node in model.graph.node if input_node.name in node.input
+    ]
+    if not all(node.op_type == "QuantizeLinear" for node in input_children):
+        return False
```

Comment:
NIT: can we put a quick note here that we're failing because the next part after an input must be a quantized block